Doc/library/imaplib.rst

```diff
@@ -78,7 +83,7 @@ There's also a subclass for secure connections:
 
 
 .. class:: IMAP4_SSL(host='', port=IMAP4_SSL_PORT, keyfile=None, \
-                     certfile=None, ssl_context=None)
+                     certfile=None, ssl_context=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT)
```

Comment:
Again, I'd use `timeout=None`. And a description of `timeout` is needed here, too.